Formula/neovim.rb

```diff
@@ -2,26 +2,11 @@ class Neovim < Formula
   desc "Ambitious Vim-fork focused on extensibility and agility"
   homepage "https://neovim.io/"
   license "Apache-2.0"
-  revision 1
 
   # Remove `stable` block when `gperf` is no longer needed.
   stable do
-    url "https://github.com/neovim/neovim/archive/v0.7.2.tar.gz"
-    sha256 "ccab8ca02a0c292de9ea14b39f84f90b635a69282de38a6b4ccc8565bc65d096"
-
-    # Libtool is needed to build `libvterm`.
-    # Remove this dependency when we use the formula.
-    depends_on "libtool" => :build
-    # GPerf was removed in https://github.com/neovim/neovim/pull/18544.
-    # Remove dependency when relevant commits are in a stable release.
-    uses_from_macos "gperf" => :build
-
-    # TODO: Use `libvterm` formula when the following is released:
-    # https://github.com/neovim/neovim/pull/17329
-    resource "libvterm" do
-      url "https://www.leonerd.org.uk/code/libvterm/libvterm-0.1.4.tar.gz"
-      sha256 "bc70349e95559c667672fc8c55b9527d9db9ada0fb80a3beda533418d782d3dd"
-    end
+    url "https://github.com/neovim/neovim/archive/v0.8.0.tar.gz"
+    sha256 "505e3dfb71e2f73495c737c034a416911c260c0ba9fd2092c6be296655be4d18"
```

Comment:
The comment above says to remove the `stable` block when `gperf` isn't needed... not sure what that means, but it looks like it isn't needed...